Fixes the process calls with nsIProcess on Windows

sfrcorne opened the pull request:
When there where spaces in the arguments of the process calls:
    -`latex_process.run`
    -`shell_process.run`
`nsIProcess` would add additional quotes around the arguments causing the process to fail.

This fix should work on both Unix and Windows but I did NOT test this on any operating system for TB 68. It should be tested on Unix and Windows, to make sure it works. (Closes #74)

sfrcorne committed:
Fixes the process calls with nsIProcess on Windows

When there where spaces in the arguments of the process calls:
    -latex_process.run
    -shell_process.run
nsIProcess would add additional quotes around the arguments causing the process to fail.

sphh commented:
Thank you so much! That's really kind of you, to also make a pull request for the master branch!

sfrcorne commented:
No problem! Let me know if there are any issues on Unix or TB 68, since I did not test these cases.

sfrcorne committed:
Changed the shell_process.run call on Unix

Changed the process calls with `shell_process.run` on Unix back to what it used before the previous commit.

sphh commented:
Thanks again! I have merged your other PR for the `mailextension`, but I'll leave this to @protz to merge, because he is the master of the `master` branch.

sphh commented:
Ok, since it works for the `mailextension` branch, I will merge this as well, so that people coming here have an up-to-date branch.
